docker-stack-wait.sh

```diff
@@ -78,20 +79,21 @@ service_state() {
   fi
 }
 print_service_logs() {
-  if [ "$opt_p" != "0" ]; then
+  if [ "$opt_l" != "0" ]; then
```

Comment:
Ahh, I see the mistake now.
Yeah, looks like I accidentally changed the default options, so it now does `docker service logs  "$service_id"`, which never stops.

Sorry about that!

Reply:
Never mind, I am glad to be able to help this project.